cmd/replicate-edit.go

```diff
@@ -185,12 +181,11 @@ func mainReplicateEdit(cliCtx *cli.Context) error {
 
 	opts := replication.Options{
 		TagString:    cliCtx.String("tags"),
-		RoleArn:      cliCtx.String("arn"),
 		StorageClass: cliCtx.String("storage-class"),
 		RuleStatus:   state,
 		ID:           cliCtx.String("id"),
 		Op:           replication.SetOption,
-		DestBucket:   cliCtx.String("remote-bucket"),
+		DestBucket:   cliCtx.String("arn"),
```

Comment:
we cannot reuse the flag with a different meaning. It will mean different in existing scripts

Reply:
why? the --arn flag already has the destination bucket name in its naming convention. Since we are dispensing with role Arn  and using it in the Destination bucket section of the replication configuration, this is precisely what we want to do here. Even if this was being used in existing scripts, this is the conversion we will make internally for backward compatibility.

Reply:
ok, added back arn flag